.env

```diff
@@ -1,5 +1,3 @@
-PUBLIC_URL=http://localhost:5000/
-
-## PREFIXED: So we can use it in CRA and Node SSR
-REACT_APP_DEFAULT_HOST=http://localhost:8080
+PUBLIC_URL=http://localhost:3000/
```

Comment:
is this being used?

Reply:
Yup: https://github.com/dotCMS/dotcms-spa-edit-mode/blob/master/public/index.html